README.md

```diff
@@ -213,6 +140,15 @@ Here are some links to documentation for this project:
 
 Understand more about our technical decisions made along with this project development in [Technical Decisions Wiki page](https://github.com/anitab-org/mentorship-backend/wiki/Technical-Decisions).
 
+### Poetry commands
+As said earlier, we use Poetry. It's simple yet powerful, but we know that many people use other package management tools.
```

Comment:
```suggestion
As said earlier, we use [Poetry](https://python-poetry.org/).
```
Small suggestion, I removed the 2nd part of the sentence to make this intro to the commands shorter. With regards to documentation, it's important to be concise right? (that is why the run step was reduced again to 1 mini section that supports both windows and linux/mac run) So there's no need to leave opinionated text 🤔 It seems like filler.

Reply:
You're right, that sentence doesn't add much value